### PR TITLE
Add missing ports in Network and Hardware requirement

### PR DESCRIPTION
**Pull request by mingshuoqiu**
Issue: https://github.com/harvester/harvester/issues/5680

### CODE REVIEW

File: docs/install/requirements.md [hunk 1]
```diff
@@ -49,8 +49,9 @@ Harvester nodes require the following port connections or inbound rules. Typical
 | Protocol  |   Port                 |  Source                                |   Description                           |
 |:----------|:---------------------------|:-----------------------------------------|:----------------------------------------|
 | TCP    |   2379                 |  Harvester management nodes            |   Etcd client port                      |
-| TCP       | 2381                     | Harvester management nodes              | Etcd health checks                    |
+| TCP       | 2381                     | Harvester management nodes              | Etcd metric collection                |
```
Comment (jillian-maroket) [inline docs/install/requirements.md:52]:
```suggestion
| TCP       | 2381                     | Harvester management nodes              | etcd metrics collection                |
```

File: docs/install/requirements.md [hunk 2]
```diff
@@ -49,8 +49,9 @@ Harvester nodes require the following port connections or inbound rules. Typical
...
 | TCP       | 2380                     | Harvester management nodes              | Etcd peer port                        |
+| TCP       | 2382                     | Harvester management nodes              | Etcd client port (http only)          |
```
Comment (jillian-maroket) [inline docs/install/requirements.md:54]:
```suggestion
| TCP       | 2382                     | Harvester management nodes              | etcd client port (HTTP only)          |
```

File: docs/install/requirements.md (hunk 1 above)
Comment (jillian-maroket) [reply docs/install/requirements.md:52]: etcd (all lowercase) is the official spelling so we must follow that.

File: docs/install/requirements.md [hunk 3]
```diff
@@ -61,6 +62,7 @@ Harvester nodes require the following port connections or inbound rules. Typical
 | TCP       | 10250                    | Harvester management and compute nodes  | Kubelet                               |
 | TCP       | 10256                    | Harvester management and compute nodes  | Kube-proxy health checks              |
 | TCP       | 10258                    | Harvester management nodes              | Cloud-controller-manager              |
+| TCP       | 10260                    | Harvester management nodes              | Cloud-controller-manager              |
```
Comment (jillian-maroket) [inline docs/install/requirements.md:65]:
```suggestion
| TCP       | 10260                    | Harvester management nodes              | cloud-controller-manager              |
```

**Review by jillian-maroket — COMMENTED** [review]
@mingshuoqiu Sorry for the delay. Looks good overall. Just please use the correct capitalization for official component names.

File: docs/install/requirements.md (hunk 3 above)
Comment (jillian-maroket) [reply docs/install/requirements.md:65]: Yes, if we are using the hyphenated component name as seen in the [Kubernetes documentation](https://kubernetes.io/docs/concepts/architecture/cloud-controller/).

**Review by bk201 — APPROVED** [review]
lgtm

**Review by starbops — APPROVED** [review]
LGTM, thank you!

**Review by jillian-maroket — APPROVED** [review]
As mentioned, etcd (all lowercase) is the official spelling so we must follow that. I'll just create another PR to fix the capitalization issues. Please merge whenever you're ready.

**Pull request merged**